case_mapping/uco/observable.py

```diff
@@ -1099,12 +1099,14 @@ def __init__(
 class FacetMobileDevice(FacetEntity):
     def __init__(
         self,
-        IMSI=None,
-        ICCID=None,
-        IMEI=None,
-        storage_capacity=None,
-        keypad_pin=None,
-        MSISDN=None,
+        IMSI: Optional[str] = None,
```

Comment:
IMSI is encoded in UCO as being on a SIM card facet or a mobile account facet.  Please move it from this Facet and update the example accordingly.

Reply:
Reference: https://github.com/ucoProject/UCO/blob/1.3.0/ontology/uco/observable/observable.ttl#L5878